packages/lesswrong/components/posts/PostsHighlight.tsx

```diff
@@ -39,10 +35,11 @@ const PostsHighlight = ({post, maxLengthWords, forceSeeMore=false, classes}: {
     ev.preventDefault();
   }, []);
   
-  return <div className={classNames(classes.root, {[classes.maxHeight]: !expanded})}>
+  return <div className={classes.root}>
     <Components.LinkPostMessage post={post} />
     <Components.ContentItemTruncated
       maxLengthWords={maxLengthWords}
+      maxHeight={!expanded}
```

Comment:
This probably shouldn't be a prop (it'd be incorrect to ever make it be something other than `!expanded`, and `expanded` is a prop already).

Reply:
(Or if it is a prop, the name makes it sound like it should be a number, similar to `maxLengthWords`, rather than a flag.)